### PR TITLE
Move windows builds from AppVeyor to CircleCI

### PR DESCRIPTION
**Pull request by evgenykochetkov**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -148,6 +168,19 @@ jobs:
...
+      name: win/default
+    steps:
+      - run: *step-configure-git-on-windows
+      - checkout
+      - run: *step-install-yarn-on-windows
+      - run:
+          name: Install Node modules
+          command: yarn
+      - run: *step-build
+      - run: *step-verify-git-clean
```
Comment (knopki) [inline .circleci/config.yml:182]: Executor image is not pinned and can be updated by third party. It's ok for windows image, but I propose to add NVM step to pin Node version. (NVM is already installed).

File: .circleci/config.yml [hunk 2]
```diff
@@ -233,6 +266,34 @@ jobs:
             - "*.dmg"
             - "*.zip"
 
+  dist-windows:
```
Comment (knopki) [inline .circleci/config.yml:269]: How to test this?
Also, nvm step here too.

**Review by knopki — CHANGES_REQUESTED** [review]
Awesome!
Windows host works unexpectedly fast.

**Review by nkrkv — APPROVED** [review]
Great!

Shouldn’t it be targeted toward `master` instead of `0.37.x`?

**Review by brusherru — APPROVED** [review]

File: .circleci/config.yml [hunk 3]
```diff
@@ -72,6 +72,13 @@ defs:
             git config --global core.autocrlf input
             git config --system core.longpaths true
 
+    step-install-node-on-windows: &step-install-node-on-windows
+      name: Install node
+      command: |
+            nvm install 12.12.0
```
Comment (knopki) [inline .circleci/config.yml:78]: But I see `.nvmrc` in the root of repository 🤔

Comment (knopki) [reply .circleci/config.yml:78]: Btw, `12.12.0` is the last pre-LTS version. LTS starts from `12.13.0` :P

**Review by knopki — APPROVED** [review]
LGTM

**Pull request merged**